src/Header.js

```diff
@@ -0,0 +1,84 @@
+import initialStoreItems from './store-items'
+import {useState} from 'react'
+
+export default function Header(props) {
+
+    const [storeItems, setStoreItems] = useState(initialStoreItems)
+    const [filterValue, setFilterValue] = useState('most-popular')
+    const [showFruit, setShowFruit] = useState(true)
+    const [showVeg, setShowVeg] = useState(true)
+
+    let filteredStoreItems = storeItems
+
+    if (filterValue === "most-popular") {
+      filteredStoreItems.sort((a, b) => a.id.localeCompare(b.id));
+    }
+  
+    if (filterValue === "lowest-price") {
+      filteredStoreItems.sort((a, b) => a.price - b.price);
+    }
+  
+    if (filterValue === "highest-price") {
+      filteredStoreItems.sort((a, b) => b.price - a.price);
+    }
+  
+    if (filterValue === "alphabetically") {
+      filteredStoreItems.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Need to look into this localeCompare, never used it before and not sure what it does.